donate/templates/payment/thank_you_master.html

```diff
@@ -35,7 +37,7 @@ <h1 class="heading heading--primary heading--bottom-margin">{% trans "Thank you
                         </svg>
                         <div class="button__label">{% trans "Twitter" %}</div>
                     </a>
-                    <a class="button button--outline button--icon js-ga-track-click" href="mailto:?&amp;subject={{ email_subject|urlencode }}&amp;body={{ request.scheme }}://{{ request.get_host }}/{% get_locale %}" data-ga-category="Social" data-ga-action="Clicked on Button" data-ga-label="Email">
+                    <a class="button button--outline button--icon js-ga-track-click" href="mailto:?&amp;subject={{ email_subject|urlencode }}&amp;body={{ page_link }}" data-ga-category="Social" data-ga-action="Clicked on Button" data-ga-label="Email">
```

Comment:
You could replace `&amp;` here too (but of course, totally optional)

Reply:
Will update them in my next PR!